Update with style guide

lucas-tortora opened the pull request:
<!---
Hints for a successful PR:
1. It is recommended that before you submit a PR to this repository, to open an issue first and assign yourself.
This way you may get inputs and discover parallel PRs to the one you want to submit.
2. In case of a big PR, consider breaking it up to smaller PRs. This will help getting it merged in an incremental process.
3. Note that a PR should have a *single* area of responsibility. If your PR does more than one thing than it should be split to several PRs!!!!!
-->

# Description

Updated Hornet docs content to match the new style guide, and migrating to Docusaurus.  Please note that this should not be merged until the Docusaurus migration is in place as mdbooks uses a different note format. 

Fixes #(issue)

## Type of change

- [x] Documentation Fix

# How Has This Been Tested?

There shouldn't be any functional changes, only documentation style.

# Checklist:

- [x] I have made corresponding changes to the documentation
- [x] I have selected the `develop` branch as the target branch

lucas-tortora committed:
* Formatted titles

lucas-tortora committed:
* Removed include statements

lucas-tortora committed:
* Converted > Notes to :::info

lucas-tortora committed:
* Fixed broken links

lucas-tortora committed:
DOCS: Edited all sections content to match the new style guide.(first iteration)

lucas-tortora committed:
DOCS: Added bulleted list in code of conduct.

lucas-tortora committed:
* Fixed broken link
* Added minor style fixes

charlesthompson3 commented on docs/CODE_OF_CONDUCT.md:
```diff
@@ -1,2 +1,68 @@
+  - Personal appearance
+  - Race
+  - Religion
+  - Sexual identity and orientation.
```
```suggestion
  - Sexual identity and orientation
```

charlesthompson3 commented on docs/CODE_OF_CONDUCT.md:
```diff
@@ -1,2 +1,68 @@
+This Code of Conduct applies within all project spaces.  It also applies when an individual is representing the project or its community in public spaces. Examples of representing a project or community include:
+  - Using an official project e-mail address
+  - Posting via an official social media account
+  - Acting as an appointed representative at an online or offline event.
```
```suggestion
  - Using an official project e-mail address.
  - Posting via an official social media account.
  - Acting as an appointed representative at an online or offline event.
```

charlesthompson3 commented on docs/CODE_OF_CONDUCT.md:
```diff
@@ -1,2 +1,68 @@
+
+All complaints will be reviewed and investigated and will result in a response that  is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident.  Project maintainers may post details of specific enforcement policies may separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
```
```suggestion
All complaints will be reviewed, investigated, and will result in a response that  is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regards to the reporter of an incident.  Project maintainers may post details of specific enforcement policies may separately.

Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
```

charlesthompson3 commented on docs/Contribution.md:
```diff
@@ -1,2 +1,33 @@
+
+## Submit a pull request
+
+Push your branch to your `HORNET` fork and open a pull request to the **develop** branch.
```
```suggestion
## How to Contribute

### Basic Setup

`HORNET` is written in [Go](https://golang.org/).

Prerequisites:

1. Setup [Go 1.14+](https://golang.org/doc/install).
2. Fork `HORNET`.
3. Test your setup by building `HORNET`:
   `go build`

## Make your Changes

Make your changes and test them sufficiently.

## Create a Commit

Commit messages should be well formatted.

You can use this as a guide:
[Conventional Commits](https://www.conventionalcommits.org)

## Submit a Pull Request

Push your branch to your `HORNET` fork and open a pull request to the **develop** branch.
```

charlesthompson3 commented on docs/faq.md:
```diff
@@ -1,42 +1,32 @@
----
 
-### Does HORNET run on the Mainnet?
+## Does HORNET run on the Mainnet?
```
```suggestion
## Does HORNET Run on the Mainnet?
```

charlesthompson3 commented on docs/getting_started/installation_steps.md:
````diff
@@ -1,108 +1,131 @@
+   sudo systemctl enable hornet.service
+   ```
+
+You can find the Hornet configuration files under the `/var/lib/hornet` directory. See more details on how to configure Hornet in the [post installation](../post_installation/post_installation.md) chapter.
````
```suggestion
You can find the Hornet configuration files under the `/var/lib/hornet` directory. You can find more details on how to configure Hornet in the [post installation](../post_installation/post_installation.md) chapter.
```

charlesthompson3 commented on docs/getting_started/installation_steps.md:
```diff
@@ -129,72 +152,85 @@ ExecStart=/opt/hornet/hornet
-### Example of systemd unit file
-Assuming the Hornet executable is extracted to `/opt/hornet` together with configuration files, please find the following example of a `systemd` unit file:
+### Example of Systemd Unit File
+The following is an example of a `systemd` unit file. If you have extracted the Hornet executable to `/opt/hornet` together with configuration files, this file should work as is.  If you have extracted the Hornet executable in another location, please review the configuration and update it accordingly
```
```suggestion
The following is an example of a `systemd` unit file. If you have extracted the Hornet executable to `/opt/hornet` together with configuration files, this file should work as is.  If you have extracted the Hornet executable in another location, please review the configuration and update it accordingly.
```

charlesthompson3 commented on docs/getting_started/nodes_101.md:
```diff
@@ -1,81 +1,71 @@
 
-When nodes detect double spends they must decide which message to consider confirmed and which one to ignore. Nodes do
-this by using consensus rules that are built into their node software using a network protocol.
+When nodes detect double spends they must decide which message to consider confirmed and which one to ignore. Nodes do this by using consensus rules that are built into their node software using a network protocol.
```
```suggestion
When nodes detect double spends, they must decide which message to consider confirmed and which one to ignore. Nodes do this by using consensus rules that are built into their node software using a network protocol.
```

charlesthompson3 commented on docs/getting_started/nodes_101.md:
```diff
@@ -1,81 +1,71 @@
-* Only when a transaction is confirmed, can nodes update their record of balances
+* Check that a transaction is not transferring more IOTA tokens than are available on the address.
+* Respond to clients' requests for their balance.
+* Once the node has confirmed a transaction with the Tangle, update the nodes record of balances . 
```
```suggestion
* Once the node has confirmed a transaction with the Tangle, update the node's record of balances . 
```

charlesthompson3 replied:
I'm going to be honest, its tricky figuring out the plural for this one: it _can_ be "node's" since we were only referring to one in the sentence before but it could also be "nodes'" since it could be _all_ nodes based on the UTXO. I think it swings more strongly towards "node's" though.

Or maybe I confused myself.

charlesthompson3 commented on docs/getting_started/nodes_101.md:
```diff
@@ -1,81 +1,71 @@
 
-> Developers are not needed to communicate with nodes using mentioned low-level API. Developers can successfully leverage [iota client libraries](https://chrysalis.docs.iota.org/libraries/overview.html) that provide a high-level abstraction to all features IOTA nodes provide, either on HTTP API level or Event API level
+:::info
+Developers do not need to communicate with nodes using mentioned low-level API. Developers can leverage the [iota client libraries](https://chrysalis.docs.iota.org/libraries/overview.html) that provide a high-level abstraction to all features IOTA nodes provide, either on HTTP API level or Event API level
```
```suggestion
Developers do not need to communicate with nodes using a mentioned low-level API. Developers can leverage the [iota client libraries](https://chrysalis.docs.iota.org/libraries/overview.html) that provide a high-level abstraction to all features IOTA nodes provide, either on HTTP API level or Event API level.
```

charlesthompson3 commented on docs/getting_started/nodes_101.md:
```diff
@@ -1,81 +1,71 @@
-* Get messages from the Tangle
+* Get tip messages.
+* Attach new messages to the Tangle.
+* Do proof of work.
```
```suggestion
* Do proof of work (POW).
```

charlesthompson3 commented on docs/getting_started/private_tangle.md:
```diff
@@ -1,55 +1,57 @@
 This is a small tutorial on how to run your own private [Chrysalis](https://chrysalis.iota.org/) tangle.
 
-We will need to set up [a coordinator](https://legacy.docs.iota.org/docs/getting-started/1.1/the-tangle/the-coordinator) and at least one additional [node](./getting_started.md) and distribute some tokens to an address that we can manage using a wallet. We will do this by running some scripts from the *private_tangle* folder that you will find in the HORNET respository.
+You will need to set up [a coordinator](https://legacy.docs.iota.org/docs/getting-started/1.1/the-tangle/the-coordinator) and at least one additional [node](./getting_started.md) and distribute some tokens to an address that you can manage using a wallet. You will do this by running some scripts from the _private_tangle_ folder that you can find in the HORNET repository.
```
```suggestion
You will need to set up [a Coordinator](https://legacy.docs.iota.org/docs/getting-started/1.1/the-tangle/the-coordinator) and at least one additional [node](./getting_started.md) and distribute some tokens to an address that you can manage using a wallet. You will do this by running some scripts from the _private_tangle_ folder that you can find in the HORNET repository.
```

charlesthompson3 commented on docs/getting_started/private_tangle.md:
```diff
@@ -1,55 +1,57 @@
+## Configuration
 
-The scripts in the *private_tangle* folder of the HORNET repository, which we will use to run the coordinator and the nodes, are preconfigured. By default they will distribute the tokens to the following address:
+The scripts in the _private_tangle_ folder of the HORNET repository, which you will use to run the coordinator and the nodes, are preconfigured. By default, they will distribute the tokens to the following address:
```
```suggestion
The scripts in the _private_tangle_ folder of the HORNET repository, which you will use to run the Coordinator and the nodes, are preconfigured. By default, they will distribute the tokens to the following address:
```

charlesthompson3 commented on docs/getting_started/private_tangle.md:
```diff
@@ -1,55 +1,57 @@
+## Start the Coordinator
 
-In the HORNET repository, change to the *private_tangle* directory and run the *run_coo_bootstrap* script. This will create all files needed to run the network and distribute the tokens to the address we configured. It will also start the coordinator.
+In the HORNET repository, change to the _private_tangle_ directory and run the `run_coo_bootstrap` script. This will create all the necessary files to run the network, distribute the tokens to the address you configured, and start the coordinator.
```
```suggestion
In the HORNET repository, change to the _private_tangle_ directory and run the `run_coo_bootstrap` script. This will create all the necessary files to run the network, distribute the tokens to the address you configured, and start the Coordinator.
```

charlesthompson3 commented on docs/getting_started/private_tangle.md:
```diff
@@ -1,55 +1,57 @@
+To start additional nodes you can use the _run_ scripts provided in the +private_tangle_ folder. You can run them on the same device, as they are preconfigured to listen on different ports.
 
-Congratulations, you are now running a private network! To monitor your coordinator, you can find the dashboard at [http://localhost:8081](http://localhost:8081). Login to the Dashboard with `admin` as username and password.
+Congratulations, you are now running a private network! You can find the dashboard at [http://localhost:8081](http://localhost:8081), and use it to monitor your coordinator. You can log in to the Dashboard using `admin` as username and password.
```
```suggestion
Congratulations, you are now running a private network! You can find the dashboard at [http://localhost:8081](http://localhost:8081), and use it to monitor your Coordinator. You can log in to the Dashboard using `admin` as username and password.
```

charlesthompson3 commented on docs/getting_started/security_101.md:
```diff
@@ -1,38 +1,44 @@
-## Securing your device
-The security of the device that's running your node is important to stop attackers from gaining access to it.
+:::info
+Please note that servers that are reachable from the Internet are a constant target from security challengers.  Please, make sure you follow minimum security essentials summarized in this article.
```
```suggestion
Please note that servers that are reachable from the Internet are a constant target from security challengers.  Please, make sure you the follow minimum security essentials summarized in this article.
```

charlesthompson3 commented on docs/getting_started/security_101.md:
```diff
@@ -1,38 +1,44 @@
-
-### Blocking unnecessary ports
-Attackers can abuse any open ports on your device.
+If you log into your device through SSH, you should take measures to protect it from unauthorized access. Many guides have been written about this subject. For more information, see [10 Steps to Secure Open SSH](https://blog.devolutions.net/2017/4/10-steps-to-secure-open-ssh). In addition to that, one can also leverage tools such as [Fail2ban](https://www.fail2ban.org/wiki/index.php/Main_Page) to ...
```
```suggestion
If you log into your device through SSH, you should take measures to protect it from unauthorized access. Many guides have been written about this subject. For more information, see [10 Steps to Secure Open SSH](https://blog.devolutions.net/2017/4/10-steps-to-secure-open-ssh). In addition to that, you can also leverage tools such as [Fail2ban](https://www.fail2ban.org/wiki/index.php/Main_Page) to further tighten your nodes security.
```

charlesthompson3 commented on docs/getting_started/security_101.md:
```diff
@@ -1,38 +1,44 @@
+Attackers can abuse any open ports on your device. To secure your device against attacks on unused open ports, you should close all ports except those that are in use.
 
-To do so, you can use a firewall. All operating systems include firewall options. By having a firewall in place you can completely block unused and unnecessary ports.
+You can use a firewall to do so. All operating systems include firewall options. By having a firewall in place you can completely block unused and unnecessary ports.
```
```suggestion
You can use a firewall to accomplish this. All operating systems include firewall options. By having a firewall in place, you can completely block unused and unnecessary ports.
```

charlesthompson3 commented on docs/getting_started/using_docker.md:
```diff
@@ -4,47 +4,52 @@ Hornet Docker images (amd64/x86_64 architecture) are available at [gohornet/horn
+## Clone the Repository
 
-Clone the repository
+Once you have completed all the installation [requirements](#requirements), you can clone the repository br by running:
```
```suggestion
Once you have completed all the installation [requirements](#requirements), you can clone the repository or by running:
```

charlesthompson3 commented on docs/getting_started/using_docker.md:
```diff
@@ -122,42 +132,53 @@ Or pull it from Docker hub (only available for amd64/x86_64):
+### Starting an Existing Hornet
 
-### Starting an existing Hornet
+You can start an existing hornet container by running:
```
```suggestion
You can start an existing Hornet container by running:
```

charlesthompson3 commented on docs/getting_started/using_docker.md:
```diff
@@ -122,42 +132,53 @@ Or pull it from Docker hub (only available for amd64/x86_64):
+## Managing a Node
+
+:::info
+Hornet uses an in-memory cache.  In order to save all data to the underlying persistent storage, it is necessary to provide a grace period of at least 200 seconds while shutting it down.:::
```
```suggestion
Hornet uses an in-memory cache.  In order to save all data to the underlying persistent storage, it is necessary to provide a grace period of at least 200 seconds while shutting it down.
:::
```

charlesthompson3 commented on docs/getting_started/using_docker.md:
```diff
@@ -122,42 +132,53 @@ Or pull it from Docker hub (only available for amd64/x86_64):
 
 ### Restarting Hornet
 
+You can restart an existing hornet container by running:
```
```suggestion
You can restart an existing Hornet container by running:
```

charlesthompson3 commented on docs/getting_started/using_docker.md:
```diff
@@ -122,42 +132,53 @@ Or pull it from Docker hub (only available for amd64/x86_64):
 
 ### Stopping Hornet
 
+You can stop an existing hornet container by running:
```
```suggestion
You can stop an existing Hornet container by running:
```

charlesthompson3 commented on docs/getting_started/using_docker.md:
```diff
@@ -122,42 +132,53 @@ Or pull it from Docker hub (only available for amd64/x86_64):
+### Displaying Log Output
 
-### Displaying log output
+You can display an existing hornet containers logs by running:
```
```suggestion
You can display an existing Hornet containers logs by running:
```

charlesthompson3 commented on docs/getting_started/using_docker.md:
```diff
@@ -122,42 +132,53 @@ Or pull it from Docker hub (only available for amd64/x86_64):
+## Removing a Container
 
-## Removing container
+You can remove an existing hornet container by running:
```
```suggestion
You can remove an existing Hornet container by running:
```

charlesthompson3 commented on docs/getting_started/using_docker_compose.md:
```diff
@@ -1,14 +1,14 @@
 # Using Docker Compose
 
-Make sure that you have read [Using Docker](using_docker.md) before continue reading. This chapter is for more advanced users. The following link provides more information about [Docker Compose]( https://docs.docker.com/compose/).
+Make sure that you have read [Using Docker](using_docker.md) before you continue reading. This chapter is for advanced users.  You can find more information on Docker Compose in the [official Docker Compose documentation]( https://docs.docker.com/compose/).
```
```suggestion
Make sure that you have read [Using Docker](using_docker.md) before you continue reading as this chapter is for advanced users.  You can find more information on Docker Compose in the [official Docker Compose documentation]( https://docs.docker.com/compose/).
```

charlesthompson3 commented on docs/post_installation/managing.md:
````diff
@@ -64,17 +66,17 @@ Hornet can be extended by plugins. Plugins are controlled via section `node` of
 ```
 
-Additionally, plugins can be controlled via [Dashboard/web interface](./post_installation.md).
+You can also control plugins can be controlled using the [Dashboard/web interface](./post_installation.md#Dashboard).
````
```suggestion
You can also control plugins using the [Dashboard/web interface](./post_installation.md#Dashboard).
```

charlesthompson3 commented on docs/post_installation/managing.md:
```diff
@@ -64,17 +66,17 @@ Hornet can be extended by plugins. Plugins are controlled via section `node` of
 
 ## Spammer
-Hornet integrates a lightweight spamming plugin that spams the network with messages. Since the IOTA network is based on Directed Acyclic Graph in which new incoming messages are connected to previous messages (tips), it is healthy for the network to maintain some level of message rate.
+Hornet integrates a lightweight spamming plugin that spams the network with messages. The IOTA network is based on Directed Acyclic Graph.  Therefore, new incoming messages are connected to previous messages (tips). It is healthy for the network to maintain some level of message rate.
```
```suggestion
Hornet integrates a lightweight spamming plugin that spams the network with messages. The IOTA network is based on Directed Acyclic Graph. Therefore, new incoming messages are connected to previous messages (tips). It is healthy for the network to maintain some level of message rate.
```

charlesthompson3 commented on docs/post_installation/managing.md:
```diff
@@ -129,30 +133,30 @@ Your node's ledger accumulates many messages, which uses a significant disk capa
-**Delta snapshot**
-A delta snapshot points to a specific full snapshot, ie. delta snapshot consists of the changes since the last full snapshot.
+#### Delta snapshots
+A delta snapshot points to a specific full snapshot, this means a delta snapshot consists of the changes since the last full snapshot.
```
```suggestion
A delta snapshot points to a specific full snapshot, meaning a delta snapshot consists of the changes since the last full snapshot.
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -1,19 +1,13 @@
-The IOTA network is a distributed network in which data is broadcasted among IOTA nodes through a gossip protocol. To be
-able to participate in a network, each node has to establish a secure connection to other nodes in the network - to its
-peer neighbors - and mutually exchange messages.
+The IOTA network is a distributed network. It uses a gossip protocol to broadcast data among IOTA nodes. To participate in a network, each node has to establish a secure connection to other nodes in the network(to its peer neighbors), and mutually exchange messages.
```
```suggestion
The IOTA network is a distributed network. It uses a gossip protocol to broadcast data among IOTA nodes. To participate in a network, each node has to establish a secure connection to other nodes in the network (to its peer neighbors), and mutually exchange messages.
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -24,21 +18,17 @@ Each time Hornet starts, the `PeerId` is written to stdout:
 
-It is recommended however to pre-generate the identity, so you can pre-communicate it to your peers before you even
-start your node and also to retain the identity in case you delete your `./p2pstore` by accident.
+However, we recommended that you pre-generate the identity.  This way you can pre-communicate it to your peers before you even start your node, and you will be able to retain the identity in case you accidentally delete your `./p2pstore`.
```
```suggestion
However, we recommended that you pre-generate the identity. This way you can pre-communicate it to your peers before you even start your node, and you will be able to retain the identity in case you accidentally delete your `./p2pstore`.
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -47,69 +37,65 @@ Your p2p public key (base58):  8CZELJwB3aBzxJgnLMvvt1FirAwNN6jif9LavYTNHCty
-> In case there already is a `./p2pstore` with another identity, Hornet will panic and tell you that you have a previous identity which does not match with what is defined via `p2p.identityPrivateKey` (
-in that case either delete the `./p2pstore` or reset the `p2p.identityPrivateKey`).
+:::info
+In case there already is a `./p2pstore` with another identity, Hornet will panic and tell you that you have a previous identity which does not match with what is defined via `p2p.identityPrivateKey` .  In this case, you should either delete the `./p2pstore` or reset the `p2p.identityPrivateKey`.
```
```suggestion
In case there already is a `./p2pstore` with another identity, Hornet will alert you that you have a previous identity which does not match with what is defined via `p2p.identityPrivateKey` .  In this case, you should either delete the `./p2pstore` or reset the `p2p.identityPrivateKey`.
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -47,69 +37,65 @@ Your p2p public key (base58):  8CZELJwB3aBzxJgnLMvvt1FirAwNN6jif9LavYTNHCty
+from the stdout when the Hornet node started up (or which was shown via the `p2pidentity` CLI tool), and your
+configured `p2p.bindAddress`. Replace the `/ip4/<ip_address>`/`/dns/<hostname>` segments with the actual information.
+
+You can find more information about `multiaddr` is available at the [libp2p docs page](https://docs.libp2p.io/concepts/addressing/).
```
```suggestion
You can find more information about `multiaddr` at the [libp2p docs page](https://docs.libp2p.io/concepts/addressing/).
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -47,69 +37,65 @@ Your p2p public key (base58):  8CZELJwB3aBzxJgnLMvvt1FirAwNN6jif9LavYTNHCty
+## Adding Node Peers
 
-### Adding node peers
+Once you know your node's own `multiaddr`, it can be exchanged with other node owners to establish a mutual peer connection. We recommended number of peer neighbors between 4-6 to get some degree of redundancy.
```
```suggestion
Once you know your node's own `multiaddr`, it can be exchanged with other node owners to establish a mutual peer connection. We recommended a number of peer neighbors between 4-6 to get some degree of redundancy.
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -47,69 +37,65 @@ Your p2p public key (base58):  8CZELJwB3aBzxJgnLMvvt1FirAwNN6jif9LavYTNHCty
 
-Once you know your node's own `multiaddr`, it can be exchanged with other node owners to establish a mutual peer
-connection. A recommended number of peer neighbors is 4-6 to get some degree of redundancy.
+## Finding neighbors
```
```suggestion
## Finding Neighbors
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -130,32 +116,24 @@ Example `peering.json`:
 
-Hornet also supports automatically finding peers through the autopeering module. We recommend to only use autopeering
-if you have at least 4 static peers to minimize service distribution in case your autopeered peers are flaky.
+Hornet also supports automatically finding peers through the _autopeering_ module.  To minimize service distribution in case your autopeered peers are flaky, we recommend to only use autopeering if you have at least 4 static peers.
```
```suggestion
Hornet also supports automatically finding peers through the _autopeering_ module. To minimize service distribution in case your autopeered peers are flaky, we recommend to only use autopeering if you have at least 4 static peers.
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -130,32 +116,24 @@ Example `peering.json`:
+
+By default, Hornet will peer up to 4 autopeered peers and initiate a gossip protocol with them. Autopeered peers are not subject to connection trimming, the same way as mutually tethered peers aren't either.
+
+### Entry node
```
```suggestion
### Entry Node
```

charlesthompson3 commented on docs/post_installation/peering.md:
```diff
@@ -130,32 +116,24 @@ Example `peering.json`:
-watermarks exist for a certain buffer number of peers to be connected, to which not necessarily a gossip protocol is
-being done.
\ No newline at end of file
+The `p2p.connectionManager.highWatermark` and `p2p.connectionManager.lowWatermark` configuration options define "watermark" points.  Watermark points can be thought of as a filling basin where if the `highWatermark` is reached, water will be drained until it reaches the `lowWatermark` again. Similarly, the connection manager within Hornet will start trimming away connections to peers if `highWate ...
```
```suggestion
The `p2p.connectionManager.highWatermark` and `p2p.connectionManager.lowWatermark` configuration options define "watermark" points.  Watermark points can be thought of as a filling basin where if the `highWatermark` is reached, water will be drained until it reaches the `lowWatermark` again. Similarly, the connection manager within Hornet will start trimming away connections to peers if `highWatermark` peers are connected until it reaches `lowWatermark` count of peers. These watermarks exist for a certain buffer number of peers to be connected, which will not necessarily be targeted by the gossip protocol.
```

charlesthompson3 commented on docs/post_installation/post_installation.md:
```diff
@@ -1,18 +1,19 @@
 # Post-installation
 
-Once Hornet is deployed, all parameters are set via configuration files.
+Once Hornet you have deployed Hornet, you can set all the parameters using configuration files.
```
```suggestion
Once you have deployed Hornet, you can set all the parameters using configuration files.
```

charlesthompson3 commented on docs/post_installation/post_installation.md:
```diff
@@ -140,19 +142,14 @@ HTTP REST API related options exists under the section `restAPI` within the `con
-If you are concerned with resource consumption, consider turning off `restAPI.powEnabled`, which makes it so that
-clients must perform Proof-of-Work locally, before submitting a message for broadcast. In case you'd like to offer
-Proof-of-Work for clients, consider upping `restAPI.powWorkerCount` to provide a faster message submission experience.
+If you are concerned with resource consumption, consider turning off `restAPI.powEnabled`.  This way, the clients must perform Proof-of-Work locally, before submitting a message for broadcast. In case you'd like to offer Proof-of-Work to clients, consider increasing the `restAPI.powWorkerCount` to provide a faster message submission experience.
```
```suggestion
If you are concerned with resource consumption, consider turning off `restAPI.powEnabled`.  This way, the clients must perform proof of work locally, before submitting a message for broadcast. In case you'd like to offer proof of work to clients, consider increasing the `restAPI.powWorkerCount` to provide a faster message submission experience.
```

charlesthompson3 replied:
So proof of work is written out as "proof of work", "Proof-of-Work", or "POW" in a lot of our docs; I changed these ones to reflect where you mentioned POW earlier in your edits so it was consistent. Don't know if you want to switch it up later but I just wanted to draw your attention to it!

charlesthompson3 commented on docs/post_installation/run_as_verifier.md:
```diff
@@ -1,29 +1,25 @@
-your findings. This is by the way the same result as when a milestone is issued by the Coordinator, which diverges from
-a consistent ledger state.
\ No newline at end of file
+After this, if your verifier node panics because of an invalid receipt, it is clear that a one of the produced receipts is not valid.  In this case, as a verifier node operator, you are invited to inform the community and the IOTA Foundation of  your findings. This is, by the way, the same result as when the Coordinator issues a milestone, which diverges from a consistent ledger state.
```
```suggestion
After this, if your verifier node panics because of an invalid receipt, it is clear that a one of the produced receipts is not valid. In this case, as a verifier node operator, you are invited to inform the community and the IOTA Foundation of  your findings. This is, by the way, the same result as when the Coordinator issues a milestone, which diverges from a consistent ledger state.
```

charlesthompson3 commented on docs/troubleshooting.md:
```diff
@@ -1,9 +1,10 @@
 ## Something went wrong?
-- **Please open a [new issue](https://github.com/gohornet/hornet/issues/new) if you detect an error or crash (or submit a PR if you have already fixed it).**
+
+Please open a [new issue](https://github.com/gohornet/hornet/issues/new) if you detect an error or crash.  You can also submit a PR if you have already fixed it. 
```
```suggestion
Please open a [new issue](https://github.com/gohornet/hornet/issues/new) if you detect an error or crash. You can also submit a PR if you have already fixed it. 
```

charlesthompson3 reviewed (commented):
Two questions/clarifications and tiny grammar edits :)

lucas-tortora committed:
Docs: Addressed @charlesthompson3 's comments

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/CODE_OF_CONDUCT.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/Contribution.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/post_installation.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/faq.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/CODE_OF_CONDUCT.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/installation_steps.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/installation_steps.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/post_installation.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/run_as_verifier.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/troubleshooting.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/nodes_101.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/nodes_101.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/nodes_101.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/nodes_101.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/private_tangle.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/private_tangle.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/private_tangle.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/private_tangle.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/security_101.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/security_101.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/security_101.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/getting_started/using_docker_compose.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/managing.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/managing.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/managing.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

Update docs/post_installation/peering.md

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

lucas-tortora committed:
Docs
* Migrated to Docusaurus.
@todo replace custom css url in docusaurus.config.js with permanent url

lucas-tortora committed:
Docs
* Improved routes

charlesthompson3 commented on docOps/docs/post_installation/peering.md:
```diff
@@ -90,7 +90,7 @@ Germany / Asia, etc.), with your allocated high watermark resources and ask for
 ::: 
 
-You can add peers using the Hornet [dashboard](../post_installation/post_installation.md#dashboard).  To do so, go to *Peers* and click on *Add Peer*.  You can also add peers on the [peering.json](./peering.md) file.
+You can add peers using the Hornet [dashboard](post_installation.md#dashboard).  To do so, go to *Peers* and click on *Add Peer*.  You can also add peers on the [peering.json](peering.md) file.
```
```suggestion
You can add peers using the Hornet [dashboard](post_installation.md#dashboard). To do so, go to *Peers* and click on *Add Peer*.  You can also add peers on the [peering.json](peering.md) file.
```

charlesthompson3 commented on docOps/docs/post_installation/run_as_a_verifier.md:
```diff
@@ -1,25 +1,25 @@
+We recommend that you use a load balanced endpoint to multiple legacy nodes for `receipts.validator.api.address` in order to obtain high availability.
+:::
+
+After this, if your verifier node panics because of an invalid receipt, it is clear that a one of the produced receipts is not valid. In this case, as a verifier node operator, you are invited to inform the community and the IOTA Foundation of  your findings. This is, by the way, the same result as when the Coordinator issues a milestone, which diverges from a consistent ledger state.
```
```suggestion
After this, if your verifier node panics because of an invalid receipt, it is clear that a one of the produced receipts is not valid. In this case, as a verifier node operator, you are invited to inform the community and the IOTA Foundation of your findings. This is, by the way, the same result as when the Coordinator issues a milestone, which diverges from a consistent ledger state.
```

charlesthompson3 reviewed (approved):
Just two small spacing things but it looks great!

lucas-tortora committed:
Apply suggestions from code review

Applied suggestions form @charlesthompson3

Co-authored-by: charlesthompson3 <74603461+charlesthompson3@users.noreply.github.com>

luca-moser committed:
adds new signer pub key and bumps warp sync adv. range for resync (#1134)

luca-moser committed:
Release 1.0.3 (#1135)

lucas-tortora committed:
Docs
* Added landing page
* Changed deploy_docs_to_gh-pages.yml action to serve Docusaurus instead of mdbooks
* Added custom iota.css
* Removed custom slug in welcome.md
* Renamed api-reference.md to api_reference.md

lucas-tortora committed:
Merge branch 'gohornet:main' into updateWithStyleGuide

lucas-tortora committed:
Docs
* Updated deploy branches in deploy_docs_to_gh-pages.yml
* Updated CNAME in deploy_docs_to_gh-pages.yml
* Updated docusaurus.config.js dark to use dark mode by default
* Updated index.js to use docusaurus.config.js variable for title and tagline
* Updated LandingPageActions/index.js texts

lucas-tortora committed:
Merge remote-tracking branch 'origin/updateWithStyleGuide' into updateWithStyleGuide

lucas-tortora committed:
Docs
* Updated deploy branches in deploy_docs_to_gh-pages.yml
* Updated CNAME in deploy_docs_to_gh-pages.yml
* Updated docusaurus.config.js dark to use dark mode by default
* Updated index.js to use docusaurus.config.js variable for title and tagline
* Updated LandingPageActions/index.js texts
* Updated from origin develop & resolved conflicts

lucas-tortora committed:
Docs
* Removed this branch for deploy_docs_to_gh-pages.yml
* Removed /hornet/ slug from docusaurus.config.js

lucas-tortora committed:
* Renamed deploy_docs_to_gh-pages.yml to deploy-docs-to-gh-pages.yml
* Renamed docOps directory to documentation
* Updated deploy-docs-to-gh-pages.yml script to use documentation directory

lucas-tortora committed:
* Renamed deploy_docs_to_gh-pages.yml to deploy-docs-to-gh-pages.yml
* Renamed docOps directory to documentation
* Updated deploy-docs-to-gh-pages.yml script to use documentation directory

lucas-tortora committed:
* Fixed edit url
* Removed branches from deploy-docs-to-gh-pages.yml action

lucas-tortora committed:
* Fixed edit url
* Removed branches from deploy-docs-to-gh-pages.yml action

lucas-tortora committed:
Merge remote-tracking branch 'origin/updateWithStyleGuide' into updateWithStyleGuide

lucas-tortora committed:
* Added specific resolutions for packages with security issues

lucas-tortora committed:
* Upgrade Docusarus version to improve mobile nav

lucas-tortora committed:
* Fixed broken links
